### ISSUE TITLE
The task content can't show on Aria2 Web Frontend(don't use https)

### ISSUE THREAD

**Issue by KwToPA** [binux/yaaw#89]
Hi 

The task content can't show on Aria2 Web Frontend
I found some errors on console 

this  (http://i.imgur.com/iIZ86ZK.png)

and this 

```
jquery-2.1.3.js:8693 GET https://token:secret@127.0.0.1:6800/jsonrpc:6800/jsonrpc?jsoncallback=jQu…xPcHRpb24iLCJwYXJhbXMiOlsidG9rZW46cGlOYXBQbGUiXX1dXQ%3D%3D&_=1460633989927 
net::ERR_CONNECTION_CLOSED
send @ jquery-2.1.3.js:8693
jQuery.extend.ajax @ jquery-2.1.3.js:8161
ariaRequest @ jsoncall.js:16
invoke @ jsoncall.js:37
invoke @ syscall.js:42
update @ rpc.js:89 js
oncall.js:46 jsonrpc disconnect!!!
```

my JSON-RPC Path is 

```
https://token:secret@127.0.01:6800/jsonrpc
```

What should I do to fix this problem?  

Thanks

**Comment by binux** [comment]
use http instead of https to visit yaaw/demo

If you want to use https with aria2, please use https aria2 jsonrpc
endpoint as well. (you may need certificate)

**Comment by KwToPA** [comment]
HI binux
I solved the problem using your method, thanks

**Comment by KwToPA** [comment]
close #89

**Comment by KwToPA** [comment]
closes #89
